src/daemon/db_util.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::blocks::Tipset;
+use crate::cli_shared::snapshot;
+use crate::db::car::forest::FOREST_CAR_FILE_EXTENSION;
+use crate::db::car::{ForestCar, ManyCar};
+use crate::utils::db::car_stream::CarStream;
+use crate::utils::io::EitherMmapOrRandomAccessFile;
+use anyhow::Context;
+use futures::TryStreamExt;
+use std::ffi::OsStr;
+use std::fs;
+use std::io;
+use std::{
+    path::{Path, PathBuf},
+    time,
+};
+use tokio::io::AsyncWriteExt;
+use tracing::info;
+use url::Url;
+use walkdir::WalkDir;
+
+pub fn load_all_forest_cars<T>(
+    store: &ManyCar<T>,
+    forest_car_db_dir: impl AsRef<Path>,
+) -> anyhow::Result<()> {
+    let forest_car_db_dir = forest_car_db_dir.as_ref();
+    if !forest_car_db_dir.is_dir() {
+        fs::create_dir_all(forest_car_db_dir)?;
+    }
+    for file in WalkDir::new(forest_car_db_dir)
+        .into_iter()
+        .filter_map(|entry| {
+            if let Ok(entry) = entry {
+                if let Some(filename) = entry.file_name().to_str() {
+                    if filename.ends_with(FOREST_CAR_FILE_EXTENSION) {
+                        return Some(entry.into_path());
+                    }
+                }
+            }
+            None
+        })
+    {
+        let car = ForestCar::try_from(file.as_path())
+            .with_context(|| format!("Error loading car DB at {}", file.display()))?;
+        store.read_only(car.into());
+        info!("Loaded car DB at {}", file.display());
```

Comment:
Let's remove this or demote it to a DEBUG message. It's exposing an implementation detail that users should not concern themselves with.